pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -37,6 +40,7 @@ const (
 	ipv6
 	dual
 
+	gkeCNIConfigFile = "/host/etc/cni/net.d/${CNI_SPEC_NAME}"
```

Comment:
Humm, would this work? Shouldn't we need something like `os.Getenv("CNI_SPEC_NAME")`?

Reply:
Done. I have cc'd my experiment process to you all. Thank you for pointing this out.